### PR TITLE
Allow reading Nomad CA/Client cert configuration

### PR DESCRIPTION
**Pull request by cipherboy**
In the Nomad secret engine, writing to `/nomad/config/access` allows users
to specify a CA certificate and client credential pair. However, these
values are not in the read of the endpoint, making it hard for operators
to see if these values were specified and if they need to be rotated.

Add `ca_cert` and `client_cert` parameters to the response, eliding the
`client_key` parameter as it is more sensitive (and should most likely
be replaced at the same time as `client_cert`).

`Signed-off-by: Alexander Scheel <alex.scheel@hashicorp.com>`

---

Resolves: #10628

### CODE REVIEW

**Review by tomhjp — APPROVED** [review]
LGTM, cc @benashz for potential impact on TFVP

**Comment by cipherboy** [comment]
@tomhjp Looks like I finally understood the test failures (Nomad not started yet was a red herring). I pushed another commit fixing the tests.

File: builtin/logical/nomad/backend_test.go [hunk 1]
```diff
@@ -449,6 +453,8 @@ func TestBackend_max_token_name_length(t *testing.T) {
 			expected := map[string]interface{}{
 				"address":               svccfg.URL().String(),
 				"max_token_name_length": tc.tokenLength,
+				"ca_cert":               "",
+				"client_cert":           "",
```
[the remark this answers is not in the document]
Comment (cipherboy) [reply builtin/logical/nomad/backend_test.go:457]: Ah, that'd be good. I'd probably just set and read them? I have no clue about setting up Nomad to actually use them and that seems like a lot more work :-)

**Review by stevendpclark — APPROVED** [review]
lgtm!

**Pull request merged**